thorlcr/graph/thgraphslave.cpp

```diff
@@ -140,10 +140,29 @@ void CSlaveActivity::setInput(unsigned index, CActivityBase *inputActivity, unsi
     else
         outLink.set(((CSlaveActivity *)inputActivity)->queryOutput(inputOutIdx));
     assertex(outLink);
+
     while (inputs.ordinality()<=index) inputs.append(NULL);
+
     inputs.replace(outLink.getClear(), index);
 }
 
+void CSlaveActivity::appendOutput(IThorDataLink *itdl)
+{
+    if (queryJob().getOptBool("TRACEROWS"))
+    {
+        const unsigned numTraceRows = queryJob().getOptInt("numTraceRows", 10);
+        outputs.append(new CTracingThorDataLink(itdl, queryHelper(), numTraceRows));
```

Comment:
@GordonSmith The missing meta information is occuring in the master branch.   All work-units (even one produced by prior builds) has missing metat information. If I revert to the 
candidate-5.4.6, the meta information is visible again.  A problem with eclwatch? Jira ticket: https://track.hpccsystems.com/browse/HPCC-14600